packages/gestalt/src/Masonry.js

```diff
@@ -541,6 +541,8 @@ export default class Masonry<T: { ... }> extends ReactComponent<Props<T>, State<
             <div // keep this in sync with renderMasonryComponent
               className="static"
               data-grid-item
+              // $FlowFixMe[prop-missing] We're assuming `columnSpan` exists
+              data-column-span={item.columnSpan}
```

Comment:
nit: `item.columnSpan ?? 1`

also, could we just update the flow type for item?